tests/test_fx/test_tracer/test_torchaudio_model/test_torchaudio_tacotron.py

```diff
@@ -0,0 +1,56 @@
+import torch
+from torchaudio.models import Tacotron2
+from utils import trace_and_compare
+import pytest
+
+
+def _get_tacotron2_model(n_mels, decoder_max_step=2000, gate_threshold=0.5):
+    return Tacotron2(
+        mask_padding=False,
+        n_mels=n_mels,
+        n_symbol=148,
+        n_frames_per_step=1,
+        symbol_embedding_dim=512,
+        encoder_embedding_dim=512,
+        encoder_n_convolution=3,
+        encoder_kernel_size=5,
+        decoder_rnn_dim=1024,
+        decoder_max_step=decoder_max_step,
+        decoder_dropout=0.1,
+        decoder_early_stopping=True,
+        attention_rnn_dim=1024,
+        attention_hidden_dim=128,
+        attention_location_n_filter=32,
+        attention_location_kernel_size=31,
+        attention_dropout=0.1,
+        prenet_dim=256,
+        postnet_n_convolution=5,
+        postnet_kernel_size=5,
+        postnet_embedding_dim=512,
+        gate_threshold=gate_threshold,
+    )
+
+
+def test_tacotron_model():
+    n_mels = 80
```

Comment:
add a decorator `@pytest.mark.skip` on this function if this test actually fails.

Reply:
otherwise, the CI will fail. We will come back to fix these skipped (failed) tests.